_blogposts/compiler/string-literal-types-in-reason.mdx

```diff
@@ -0,0 +1,215 @@
+---
+author: hongbo
+date: "2020-07-28 18:00"
+previewImg:
+category: compiler
+title: Introducing string literal types in BuckleScript version 8.2
+description: |
+  Highlights of our newest changes to the internal representation 
+  and how they will benefit our users.
+---
+
+---
+
+## String literal types in BuckleScript
+
+[String literal types](https://www.typescriptlang.org/docs/handbook/advanced-types.html#string-literal-types) was introduced by TypeScript to model
+JavaScript behavior, it's a relatively new concept since most type systems are runtime encoding agnostic. However, to smooth the user experience when writing
+bindings to existing JS API, we are introducing string literal types which is unique in several behaviors compared with TypeScript: it supports type inference,
+pattern match and can be attached to data.
```

Comment:
Because "string literal types" is plural, change to: "...string literal types, which are unique in several behaviors when compared with Typescript: they support type inference, they can be used in pattern matching, and they can be attached to data."  (Either that, or change to "...introducing the string literal type..." and keep everything else singular.